contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -129,12 +134,16 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
       int256 fallbackNativePerUnitLink,
       uint32 requestTimeoutSeconds,
       uint96 donFee,
-      uint16 maxSupportedRequestDataVersion
-    ) = abi.decode(config, (uint32, uint32, uint32, uint32, int256, uint32, uint96, uint16));
+      uint16 maxSupportedRequestDataVersion,
+      uint256 fulfillmentGasPriceOverEstimationBP
+    ) = abi.decode(config, (uint32, uint32, uint32, uint32, int256, uint32, uint96, uint16, uint256));
 
     if (fallbackNativePerUnitLink <= 0) {
       revert InvalidLinkWeiPrice(fallbackNativePerUnitLink);
     }
+    if (fulfillmentGasPriceOverEstimationBP < 10_000) {
```

Comment:
We do. This check is for revert if _less than_ 10k, which is the 0 for basis points.

Reply:
This check is not correct any more (and not required?)

Reply:
We do. My above comment should have said 10_000 = 0 BP.
BP must always be at least 10_000, otherwise it is negative

Reply:
I am confused by this.  Why can't it just be a simple percentage value?  Users are likely to be confused as well.  I think a simple percentage is plenty granular for overestimates and is very easy for everyone to understand.

Reply:
> Why can't it just be a simple percentage value?
EIP1559's max change is 12.5%, so that already isn't granular enough
I'd rather err on the side of granularity using 1/1000th of a percent. I don't think users need to understand this super deeply to use the product.